tests/nightly/prompting-client-integration-tests/task.yaml

```diff
@@ -68,6 +68,12 @@ prepare: |
 
     pushd prompting-client/prompting-client
 
+    echo "Enable timeouts for integration tests"
+    git fetch
+    git config user.email 'nonexistent@ubuntu.com'
```

Comment:
This could be anything@example.org which is designed for ... well, examples like that.